index.html

```diff
@@ -17,63 +17,74 @@
     <div class="logo">
       <a href='#'>John Doe</a>
     </div>
-    <nav>
-      <input type="checkbox" class="check">
+    <nav >
+     
       <div class="nav-toggler">
         <div></div>
       </div>
-      <div class="menu">
-        <div>
+     
+      
           <ul class="list-item">
             <li><a href='#'>Portfolio</a></li>
             <li><a href='#'>About</a></li>
             <li><a href='#'>Contact</a></li>
+            <li><a href='#'><img src="img/Union-nav.png"></a></li>
           </ul>
-        </div>
-      </div>
+      
+    
     </nav>
   </header>
   <section class="main">
-    <h1>Hey There.I’m Alex</h1>
+    <div class="main-header">
+    <h1>Hey There.<br>I’m Alex</h1>
     <h3>I’am a Software Developer</h3>
     <p>I can help you build a product , feature or website Look through some of my work and experience!
       If you like what you see and have a project you need coded,
       don’t hestiate to contact me.</p>
-
+    </div>
+    <div class="soaial-wriper">
     <ul class="social-icon">
       <li class="social"><a href="#"><img src="img\github.png"></a></li>
       <li class="social"><a href="#"><img src="img\linkedin.png"></a></li>
       <li class="social"><a href="#"><img src="img\Vector2.png"></a></li>
       <li class="social"><a href="#"><img src="img\twitter.png"></a></li>
       <li class="social"><a href="#"><img src="img\medium.png"></a></li>
     </ul>
+    </div>
   </section>
```

Comment:
Kindly check and making sure that the header and headline section is working according to the figma design. At the moment the text and making image are scrambled together making it difficult to read and understand what you are trying to put across. :1234: There is also an empty white space created on the right side which is not suppose to be there.

Your design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-09-29 20-49-43](https://user-images.githubusercontent.com/65233860/193141376-c9ceb380-50f0-4dcd-9425-04c835613c83.png)  |  ![Screenshot from 2022-09-29 21-01-33](https://user-images.githubusercontent.com/65233860/193141541-fdb30fa4-1b5e-42ab-b4eb-ec6317a14025.png)